### PR TITLE
JBIDE-27875: use httpClient 4 from TP

### PR DESCRIPTION
**Pull request by sbouchet**
Signed-off-by: Stephane Bouchet <sbouchet@redhat.com>

### CODE REVIEW

File: jmx/plugins/org.jboss.tools.jmx.jolokia/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -23,7 +25,6 @@ Import-Package: org.eclipse.ui.views.properties.tabbed
 Bundle-Activator: org.jboss.tools.jmx.jolokia.internal.Activator
 Bundle-ClassPath: .,
  lib/commons-logging-1.1.1.jar,
- lib/httpclient-4.5.2.jar,
  lib/httpcore-4.4.5.jar,
```
Comment (jeffmaury) [inline jmx/plugins/org.jboss.tools.jmx.jolokia/META-INF/MANIFEST.MF:28]: Why this one is not removed ?

File: jmx/plugins/org.jboss.tools.jmx.jolokia/build.properties [hunk 2]
```diff
@@ -8,7 +8,6 @@ bin.includes = plugin.xml,\
                plugin.properties,\
                image/,\
                lib/commons-logging-1.1.1.jar,\
-               lib/httpclient-4.5.2.jar,\
                lib/httpcore-4.4.5.jar,\
```
Comment (jeffmaury) [inline jmx/plugins/org.jboss.tools.jmx.jolokia/build.properties:11]: Same question ?

File: jmx/plugins/org.jboss.tools.jmx.jolokia/pom.xml [hunk 3]
```diff
@@ -19,7 +19,7 @@
 	<properties>
 		<jolokia.client.version>1.4.0</jolokia.client.version>
 		<json.simple.version>1.1.1</json.simple.version>
-		<httpcomponents.version>4.5.2</httpcomponents.version>
+		<httpcomponents.version>4.5.13</httpcomponents.version>
```
Comment (jeffmaury) [inline jmx/plugins/org.jboss.tools.jmx.jolokia/pom.xml:22]: I'm lost why do you keep this and copy dependencies if taken from TP ?

File: jmx/plugins/org.jboss.tools.jmx.jolokia/META-INF/MANIFEST.MF (hunk 1 above)
Comment (sbouchet) [reply jmx/plugins/org.jboss.tools.jmx.jolokia/META-INF/MANIFEST.MF]: fixed

File: jmx/plugins/org.jboss.tools.jmx.jolokia/build.properties (hunk 2 above)
Comment (sbouchet) [reply jmx/plugins/org.jboss.tools.jmx.jolokia/build.properties]: fixed

File: jmx/plugins/org.jboss.tools.jmx.jolokia/pom.xml (hunk 3 above)
Comment (sbouchet) [reply jmx/plugins/org.jboss.tools.jmx.jolokia/pom.xml]: removed.

**Review by jeffmaury — APPROVED** [review]
LGTM

**Pull request merged**